csharp-advanced-topics/Delegates/DelegateTests/DelegateTests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
```

Comment:
.NET 5 is good, but .NET 6 is better. You don't need to change it now, but if we go to the next phase .NET 6 is mandatory for all the articles in order to keep them fresh for longer.